src/theme/Theme.scss

```diff
@@ -78,3 +78,7 @@ $size-values: (
   m: 32px,
   l: 48px
 );
+
+//Animations
+
+$animation-1: 0.3s
```

Comment:
Definiraj kaj se animira, lahko das da vse, kak dolgo in uporabi ease-in-out nacin

Reply:
Tocno tak. Da definiras kaj se animira, kak dolgo in na kaksen nacin. Ce se boma kje animacije uporabljala. Uporabis kar to.